assets/app/views/project.html

```diff
@@ -54,29 +56,27 @@ <h2 class="service" ng-init="numPorts = service.spec.ports.length">{{serviceId}}
                   <!-- TODO right now we ONLY show deployments / RCs if they have pods, but we may also want to show 
                        them if they are idled down to zero for example.
                   -->
-                  <div ng-if="(podsByDeployment[deployment.metadata.name] | hashSize) > 0">
-                    <div class="small muted" ng-if="deployment" style="margin-bottom: 10px;">
-                      <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp>
-                      <span ng-if="deployment.details && deployment.details.causes && deployment.details.causes.length > 0">
-                        <span>, triggered by 
-                          <span class="deployment-trigger" ng-repeat="cause in deployment.details.causes">
-                            <span ng-switch="cause.type">
-                              <span ng-switch-when="ImageChange">new image for {{cause.imageTrigger.repositoryName | imageName}}:{{cause.imageTrigger.tag}}</span>
-                              <span ng-switch-when="ConfigChange">deployment configuration change</span>
-                            </span>
+                  <div class="small muted" ng-if="deployment" style="margin-bottom: 10px;">
+                    <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp>
+                    <span ng-if="deployment.details && deployment.details.causes && deployment.details.causes.length > 0">
+                      <span>, triggered by 
+                        <span class="deployment-trigger" ng-repeat="cause in deployment.details.causes">
+                          <span ng-switch="cause.type">
+                            <span ng-switch-when="ImageChange">new image for {{cause.imageTrigger.repositoryName | imageName}}:{{cause.imageTrigger.tag}}</span>
+                            <span ng-switch-when="ConfigChange">deployment configuration change</span>
                           </span>
                         </span>
                       </span>
-                    </div>
-                    <div style="display: inline-block;">
-                      <!-- TODO figure out why podTemplate can't be done the same way as pods -->
-                      <pod-template ng-init="podTemplate = deployment.spec.template"></pod-template>
-                      <pods pods="podsByDeployment[deployment.metadata.name]"></pods>
-                    </div>
+                    </span>
+                  </div>
+                  <div style="display: inline-block;">
+                    <!-- TODO figure out why podTemplate can't be done the same way as pods -->
+                    <pod-template ng-init="podTemplate = deployment.spec.template"></pod-template>
+                    <pods pods="podsByDeployment[deployment.metadata.name]"></pods>
```

Comment:
yes, it does an ng-repeat over pods, ng-repeats handle undefined/empty,  in that case it will just render an empty `<div class="flex-row">`